storage/files.rb

```diff
@@ -257,76 +257,127 @@ def copy_file project_id:, source_bucket_name:, source_file_name:,
   # [END copy_file]
 end
 
+def rotate_encryption_key project_id:, bucket_name:, file_name:,
+                          current_encryption_key:, new_encryption_key:
+  # [START rotate_encryption_key]
+  # project_id             = "Your Google Cloud project ID"
+  # bucket_name            = "Your Google Cloud Storage bucket name"
+  # file_name              = "Name of a file in the Cloud Storage bucket"
+  # current_encryption_key = "Encryption key currently being used"
+  # new_encryption_key     = "New encryption key to use"
+
+  require "google/cloud/storage"
+
+  storage = Google::Cloud::Storage.new project: project_id
+  bucket  = storage.bucket bucket_name
+  file    = bucket.file file_name, encryption_key: current_encryption_key
+
+  file.rotate encryption_key: current_encryption_key,
```

Comment:
Minor: let's align the values of these 2 parameters with one another for consistency.